templates/docs/document.html

```diff
@@ -1,6 +1,8 @@
 
 {% extends "docs/base.html" %}
 
+{% block meta_title %}Snapcraft documentation | {{ document.title }} {% endblock %}
```

Comment:
I think I'd do it opposite: "Document title | Snapcraft documentation" ... as title seems more relevant to what people want
```suggestion
{% block meta_title %}{{ document.title }} | Snapcraft documentation{% endblock %}
```